deployment/deployment.js

```diff
@@ -1,240 +1,25 @@
-    );
-  } else if (args.protocol in protocolNetworkMap[args.subgraph] == false) {
-    console.log(
-      "Error: please specify a valid protocol as 1st argument or add to configurations  (i.e. uniswap-v2, sushiswap, etc.)"
-    );
-    console.log(
-      "To deploy all networks of a specified protocol, pass 2 arguements (protocol/location)"
-    );
-  } else {
-    let protocol = args.protocol;
-
-    for (const network in protocolNetworkMap[args.subgraph][protocol]) {
-      let template =
-        protocolNetworkMap[args.subgraph][protocol][network]["template"];
-      let location = "";
-      let prepareConstants =
-        protocolNetworkMap[args.subgraph][protocol][network][
-          "prepare:constants"
-        ];
-
-      // Get location for configurations or derive using standard naming convention
-      if (
-        args.location in protocolNetworkMap[args.subgraph][protocol][network]
-      ) {
-        location =
-          protocolNetworkMap[args.subgraph][protocol][network][args.location];
-      } else {
-        location =
-          args.location + "/" + protocol + "-" + getDeploymentNetwork(network);
-      }
-
-      // Don't execute the script if the location is messari, you have specified not to deploy on merge, and this script is excuted by Github Action upon merge
-      if (
-        args.location == "messari" &&
-        [false, undefined].includes(
-          protocolNetworkMap[args.subgraph][protocol][network][
-            "deploy-on-merge"
-          ]
-        ) &&
-        ["true", "t"].includes(args.merge.toLowerCase()) &&
-        args.deploy != "build"
-      ) {
-        results += "Ignored in Deployment Configurations: " + location + "\n";
-      } else {
-        allScripts.set(
-          location,
-          scripts(
-            protocol,
-            network,
-            template,
-            location,
-            prepareConstants,
-            args.type
-          )
-        );
-      }
-    }
-
-    runCommands(allScripts, results, args, function (results) {});
-  }
-} else if (args.subgraph) {
-  if (args.subgraph in protocolNetworkMap == false) {
-    console.log(
-      "Error: please specify a a valid subgraph directory or add to configurations  (e.g. uniswap-forks, compound-forks, qidao, etc"
-    );
-  } else {
-    for (const protocol in protocolNetworkMap[args.subgraph]) {
-      for (const network in protocolNetworkMap[args.subgraph][protocol]) {
-        let template =
-          protocolNetworkMap[args.subgraph][protocol][network]["template"];
-        let location = "";
-        let prepareConstants =
-          protocolNetworkMap[args.subgraph][protocol][network][
-            "prepare:constants"
-          ];
+  throw "Please check package.json scripts in local subgraph folder. This error is being thrown becuase it is missing a parameteter in the 'deploy' script. You can find an updated version of the scripts in the deployments folder at the head of the directory.";
```

Comment:
thoughts on having some sort of logic to tell which item is missing (can be the first one it runs into) then notifies that? Maybe not worth the effort though